src/test/RASTER_IMAGE_DATA_PERFORMANCE.test.ts

```diff
@@ -0,0 +1,383 @@
+            
+                    eventDataTx.set(Utility.stringToUint8Array("FILE_LIST_REQUEST", 32));
+                    eventDataTx.set(new Uint8Array(new Uint32Array([1]).buffer), 32);
+                    eventDataTx.set(payload, 36);
+            
+                    Connection.send(eventDataTx);
+            
+                    // While receive a message
+                    Connection.onmessage = (event: MessageEvent) => {
+                        let eventName = Utility.getEventName(new Uint8Array(event.data, 0, 32));
+                        if (eventName === "FILE_LIST_RESPONSE") {
+                            expect(event.data.byteLength).toBeGreaterThan(0);
+                            let eventData = new Uint8Array(event.data, 36);
+                            expect(CARTA.FileListResponse.decode(eventData).success).toBe(true);
+            
+                            done();
+                        }
+                    };
+                }, openFileTimeout);
+            }
+        );
+    });
+
+    describe(`prepare the files`, () => {
+        [ 
+         ["cluster_00128.fits", {xMin: 0, xMax:   128, yMin: 0, yMax:   128},  1, CARTA.CompressionType.ZFP, 18, 4],
+         ["cluster_00256.fits", {xMin: 0, xMax:   256, yMin: 0, yMax:   256},  1, CARTA.CompressionType.ZFP, 12, 4],
+         ["cluster_00512.fits", {xMin: 0, xMax:   512, yMin: 0, yMax:   512},  2, CARTA.CompressionType.ZFP, 11, 4], 
+         ["cluster_01024.fits", {xMin: 0, xMax:  1024, yMin: 0, yMax:  1024},  3, CARTA.CompressionType.ZFP, 11, 4],
+         ["cluster_02048.fits", {xMin: 0, xMax:  2048, yMin: 0, yMax:  2048},  6, CARTA.CompressionType.ZFP, 11, 4],
+         ["cluster_04096.fits", {xMin: 0, xMax:  4096, yMin: 0, yMax:  4096}, 12, CARTA.CompressionType.ZFP, 11, 4],  
+         ["cluster_08192.fits", {xMin: 0, xMax:  8192, yMin: 0, yMax:  8192}, 23, CARTA.CompressionType.ZFP, 11, 4],
+         ["cluster_16384.fits", {xMin: 0, xMax: 16384, yMin: 0, yMax: 16384}, 46, CARTA.CompressionType.ZFP, 11, 4],
+         ["cluster_32768.fits", {xMin: 0, xMax: 32768, yMin: 0, yMax: 32768}, 92, CARTA.CompressionType.ZFP, 11, 4],
+         ["hugeGaussian10k.fits", {xMin: 0, xMax: 10000, yMin: 0, yMax: 10000},  28, CARTA.CompressionType.ZFP, 11, 4],
+         ["hugeGaussian20k.fits", {xMin: 0, xMax: 20000, yMin: 0, yMax: 20000},  56, CARTA.CompressionType.ZFP, 11, 4],
+         ["hugeGaussian40k.fits", {xMin: 0, xMax: 40000, yMin: 0, yMax: 40000}, 112, CARTA.CompressionType.ZFP, 11, 4],
+         ["hugeGaussian80k.fits", {xMin: 0, xMax: 80000, yMin: 0, yMax: 80000}, 223, CARTA.CompressionType.ZFP, 11, 4],
+        ].map(
+            function ([testFileName, imageBounds, mip, compressionType, compressionQuality, numSubsets]: 
+                    [string, {xMin: number, xMax: number, yMin: number, yMax: number}, number, CARTA.CompressionType, number, number]) {
+                test(`assert the file "${testFileName}" loads info.`, 
```

Comment:
same question here: We've already got tests that check for file info. @kswang1029 thoughts on this?

Reply:
I suggest in this performance test we do not make assertions on file info. File info assertions should be in the dedicated test. At this moment I only add very basic file info assertions. Once we have settle down the backend, I will add more. Currently the two BEs generate a bit different file info.

Reply:
About the performance testing, do we need to have the timeout function to postpone the test after a previous test? In case the tests in my laptop have unstable outcome of performance. The disk IO supposes to be the issue.

Reply:
we should prevent multiple tests from running at the same time. Ideally, we should find a way to clear disk cache between tests as well.